guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.md

```diff
@@ -0,0 +1,115 @@
+---
+group: software-update-guide
+title: Manage Inventory Management modules
+---
+
+Inventory Management modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in Magento 2.3.X Open Source and Commerce.
+
+These features and extensions were developed as part of the [Multi Source Inventory (MSI) project](https://github.com/magento-engcom/msi) through the Magento Community Engineering program.
+
+## Installation
```

Comment:
The other headings start with verbs, so maybe change this to "Install Inventory Management".